src/protocol_pro_2.cpp

```diff
@@ -41,7 +42,53 @@ Pro2ProtocolObject::Pro2ProtocolObject(
       std::thread([this]() { this->motors_control_loop(30); });
 }
 
-void Pro2ProtocolObject::update_drivetrim(double value) { trimvalue_ += value; }
+void Pro2ProtocolObject::get_params_file() {
+  // open yaml
```

Comment:
small comment here but can we move to always using /* comment */ style instead of // ; even for single line comments. It's just easier to read IMO